content/theme/minimal-bootstrap-blog.md

```diff
@@ -0,0 +1,33 @@
+---
+title: "minimal-bootstrap-blog"
+github: https://github.com/andrewhwanpark/minimal-bootstrap-blog
+demo: https://andrewhwanpark.github.io/minimal-bootstrap-blog/ # Please make sure this links to a theme demo and not your personal/business site. The demo url must use https
+author: andrewhwanpark
+date: 2021-07-20 # Enter the date you submitted the theme YYYY-MM-DD
+ssg:
+  - Jekyll # at least 1 ssg is required
+cms:
+  - No CMS # at least 1 cms is required. If your theme uses markdown (no CMS) the cms should be "No CMS"
+css: # css is optional
+  - Bootstrap-dark-5
```

Comment:
Bootstrap-dark-5 isnt valid - please change to `Bootstrap`

Reply:
Thanks!